ai-service/src/ai_service/embeddings/embedder.py

```
@@ -0,0 +1,138 @@
+
+def _get_device() -> str:
+    """
+    Auto-detect the best available device for embedding computation.
+
+    Returns:
+        Device string: 'cuda', 'mps', or 'cpu'
+    """
+
+    if torch.cuda.is_available():
+        return "cuda"
+    elif hasattr(torch.backends, "mps") and torch.backends.mps.is_available():
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)